pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -27,7 +27,7 @@ style_min: true
   - if request.language == "en"
     = view :curriculum_banner
     = view :homepage_below_hero_announcement
-    = view :homepage_below_hero_plane_banner
+    = view :homepage_below_hero_trophy_banner
```

Comment:
Glad I left the partial in the source tree.  Had a feeling we'd need it again :)

Reply:
Yes, thank you made it very easy!